README.md

```diff
@@ -98,3 +106,57 @@ specify it on every invocation of cargo test:
 ```bash
 CLASSPATH="/path/to/jna-5.14.0.jar:/path/to/kotlinx-coroutines-core-jvm-1.6.4.jar" cargo test
 ```
+
+## Local Development
+
+### Kotlin
+
+To locally test integration with `mobile-sdk-kt`, it is preferrable to use a `mavenLocal()` repository.
+
+To release to `mavenLocal()` you may use the following command:
+
+```bash
+cd kotlin/ && VERSION=x.y.z ./gradlew publishDebugPublicationToMavenLocal
+```
+
+Where `VERSION` is set to a SemVer (Semantic Versioning). Note that it is possible to use a tagged version, e.g. `0.0.33-SNAPSHOT`, which may be preferrable to denote
```

Comment:
I might make this language even stronger. Something like "It is strongly advised to use a tagged version, as using a regular version can cause confusion down the line when the local maven repository contains conflicting version numbers." (Yes, this has bit me in the past).